ACPs/77-reinventing-subnets/README.md

```diff
@@ -32,48 +32,194 @@ With no Elastic Subnets live on Mainnet, it is clear that Permissionless Subnets
 
 ## Specification
 
-### Separate Subnet Validators from Primary Network Validators
+At a high-level, Subnets will now be able to manage their validator sets external to the P-Chain by setting the blockchain ID and address of their "validator manager". This validator manager is able to send Avalanche Warp Messages to the P-Chain in order to modify the validator set registered for the Subnet. In response, the P-Chain will send Avalanche Warp Message back to validator managers to notify them of validator set changes that have taken effect. Subnet Validators will no longer be required to validate the Primary Network, which removes the requirement to stake 2000 $AVAX to be a Subnet Validator. Further, Subnet Validators will only be required to sync the P-Chain (not X/C-Chain) in order to track validator set changes and support cross-Subnet communication. To maintain a Subnet Validator on the P-Chain, a continuous fee must be paid in $AVAX as specified later in this ACP.
```

Comment:
nit: why is Validator capitalized in some places but not others, is it solely for "Subnet Validators"?

Reply:
Yeah exactly, it has only been for "Subnet Validators". I'm not 100% sure if it should even be capitalized then, but I think it's still consistent to that end

Reply:
I made all mentions of validator lower case in #148